project1/application.py

```diff
@@ -25,7 +25,19 @@
 def index():
     return "Project 1: TODO"
 
+
 @app.route("/data/batch")
 def codetime():
     print(request.data)
     return "Codetime Success!!"
+
+
+@app.route("/register")
+def register():
+    return render_template('register.html')
+
+
+@app.route("/hello", methods=["POST"])
+def hello():
+    name = request.form.get("Email")
+    pwd = request.form.get("password")
```

Comment:
both has to be bought under 1 route /register with explicitly check for which method request is made from client based on that perform action. Ex: If and Else blocks for GET and POST.

Reply:
updated in the latest version sir.